package.json

```
@@ -72,6 +72,6 @@
     "yorkie": "2.0.0"
   },
   "engines": {
-    "node": "^16.20.0 || ^18.0.0 || ^20.0.0 || ^22.0.0"
+    "node": ">=16.20.0"
```

This isn't specific enough, we really need something more like this:

```suggestion
    "node": "^16.20.0 || ^18.0.0 || ^20.0.0 || >=22.0.0"
```